draft-ietf-quic-http.md

```diff
@@ -794,6 +815,37 @@ the server MAY send another GOAWAY frame with an updated last stream identifier.
 This ensures that a connection can be cleanly shut down without losing requests.
 
 
+# MAX_PUSH_ID {#frame-max-push-id}
+
+The MAX_PUSH_ID frame (type=0xA) is used by clients to control the number of
```

Comment:
Note that you're now conflicting with the frame type for the ALTSVC frame in the HTTP/2 registry.  ORIGIN is using 0xC, which suggests someone has 0xB but I can't think of who off the top of my head right now.  Might want to jump out to at least 0xD to be safe, or (ab)use 0x8 instead.